stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -508,6 +531,12 @@
     (and (>= lock-period MIN_POX_REWARD_CYCLES)
          (<= lock-period MAX_POX_REWARD_CYCLES)))
 
+;; Is the given burn block height in the prepare phase?
```

Comment:
Can you add this comment, so other folks reading this have an easier time following it?

```
;; This computes `((height - first-burnchain-block-height) + pox-prepare-cycle-length) % pox-reward-cycle-length) < pox-prepare-cycle-length`.
```

Reply:
Also, you can save one read-count if you store `pox-prepare-cycle-length` with a `let`-binding :)